test/test/otlp/OtlpTests.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package test.otlp;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import one.profiler.test.*;
+import io.opentelemetry.proto.profiles.v1development.*;
+
+public class OtlpTests {
+    @Test(mainClass = CpuBurner.class, agentArgs = "start,otlp,file=%f.pb")
+    public void testOtlpReadable(TestProcess p) throws Exception {
+        ProfilesData profilesData = waitAndGetProfileData(p);
```

Comment:
ProfilesData or ProfileData?

Reply:
I know its `ProfilesData` in the protocol. The question was about the function called `*GetProfile_Data` that returns `Profile_s_Data`

Reply:
Yeah I noticed, I fixed it here: 30aa65327137ce7de5f791cbf7a8d62b64a733cc